imgui-binding/src/main/java/imgui/extension/implot/ImPlotLimits.java

```diff
@@ -0,0 +1,89 @@
+package imgui.extension.implot;
+
+import imgui.binding.ImGuiStruct;
+
+public final class ImPlotLimits extends ImGuiStruct {
+    public ImPlotLimits(final long ptr) {
+        super(ptr);
+    }
+
+    public ImPlotLimits() {
+        this(0, 0, 0, 0);
+    }
+
+    public ImPlotLimits(final double xMin, final double xMax, final double yMin, final double yMax) {
+        this(create(xMin, xMax, yMin, yMax));
+    }
+
+    /*JNI
+        #include "_common.h"
+        #include "_implot.h"
+
+     */
+
+    private static native long create(double xMin, double xMax, double yMin, double yMax); /*
+        return (intptr_t)(new ImPlotLimits(xMin, xMax, yMin, yMax));
+    */
+
+    public boolean contains(final ImPlotPoint plotPoint) {
+        return contains(plotPoint.getX(), plotPoint.getY());
+    }
+
+    public boolean contains(final double x, final double y) {
+        return nContains(this.ptr, x, y);
+    }
+
+    private static native boolean nContains(long ptr, double x, double y); /*
+        return ((ImPlotLimits*)ptr)->Contains(x, y);
+    */
+
+    public ImPlotPoint min() {
+        return new ImPlotPoint(nMin(this.ptr));
+    }
+
+    private static native long nMin(long ptr); /*
+        ImPlotPoint* p = new ImPlotPoint(((ImPlotLimits*)ptr)->Min());
```

Comment:
It seems like in all places with calling `new` you'll create a memory leak, since in C++ we need to deallocate the heap memory.